docs/infrastructure.md

```diff
@@ -172,279 +170,213 @@ When you launch the graph backend for the first time,
-    curl -X PUT -u "admin:NewAdminPassword" http://localhost:7200/repositories/my_db --data-binary "@data-config.ttl" -H "Content-Type: application/x-turtle"
+    Done.
     ```
 
-    and add give our user access permission to the new resource:
+    You can now proceed to the section [Uploading data to the graph](#uploading-data-to-the-graph).
+
+=== "Stardog"
+
+    **Note: Stardog has been deprecated as a supported Neurobagel graph backend.**
+
+## Updating your graph backend configuration
+
+### Updating existing database user permissions
 
+!!! tip
+    You can verify a user's settings at any time with the following:
     ```bash
-    curl -X PUT --header 'Content-Type: application/json' -d '
-    {"grantedAuthorities": ["WRITE_REPO_my_db","READ_REPO_my_db"]}'  http://localhost:7200/rest/security/users/DBUSER -u "admin:NewAdminPassword"
+    curl -u "admin:NewAdminPassword" http://localhost:7200/rest/security/users/DBUSER
     ```
 
-    - `"WRITE_REPO_my_db"`: Grants write permission.
-    - `"READ_REPO_my_db"`: Grants read permission.
+If you want to change database access permissions (e.g., adding or removing access to a database) for an _existing_ user in your GraphDB instance, you must do so manually.
 
-    !!! Note
-    
-        Make sure you replace `my_db` with the name of the graph db you 
-        have just created. 
+Of note, in GraphDB, there is no straightforward REST API call to update a user's database access permissions without replacing the list of their existing database permissions (`"grantedAuthorities"`) entirely. 
 
-    !!! warning "Be careful when adding more user permissions"
+For example, if user `DBUSER` has been granted read/write access to database `my_db1` with the following command:
 
-        With GraphDB, there is no straightforward REST API call to _update_ a user's database access permissions without replacing the list of their existing database permissions (`"grantedAuthorities"`) entirely. 
-        
-        For example, if user `DBUSER` has been granted read/write access to database `my_db1` via a curl command with the following:
-        `{"grantedAuthorities": ["WRITE_REPO_my_db1","READ_REPO_my_db1"]}`
+```bash
+curl -X PUT --header 'Content-Type: application/json' -d '
+{"grantedAuthorities": ["WRITE_REPO_my_db","READ_REPO_my_db"]}' http://localhost:7200/rest/security/users/DBUSER -u "admin:NewAdminPassword"
+```
 
-        To grant `DBUSER` read/write access to a second database `my_db2` (while keeping the existing access to `my_db1`), 
-        when modifying http://localhost:7200/rest/security/users/DBUSER, 
-        you must specify all permissions - existing and new - in the same command 
-        since the existing permissions list will be overwritten:
-        `{"grantedAuthorities": ["WRITE_REPO_my_db1","READ_REPO_my_db1", "WRITE_REPO_my_db2","READ_REPO_my_db2"]}`
+!!! info
+    This command is also run as part of the [`graphdb_setup.sh`]() script for first-time GraphDB backend setup.
```

Comment:
Visually distracting for how important this is (already have many boxes). If you think this is important, I'd put it in brackets in the freeform text:

For example, if user `DBUSER` has been granted read/write access to database `my_db1` with the following command + **(this command is run by default when you use `graphdb_setup.sh`)**